test/e2e/main_test.go

```diff
@@ -17,6 +17,7 @@ import (
 )
 
 var coAPIPresent = true
+var ProxyAPIPresent = true
```

Comment:
Why do you need this variable?

Reply:
Good point - I had meant to remove this. I wanted to run the proxy tests part of the `OperatorGroup` tests but ran into a circular dependency issue. I instead exported a function in the `testgroups` package that lets you set a variable to run the proxy tests.